src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -585,6 +594,14 @@ class DocBaseViewer extends BaseViewer {
             };
         }
 
+        // If range requests are disabled, request the gzip compressed version of the representation
+        if (PDFJS.disableRange) {
+            const { url } = docInitParams;
+            docInitParams.url = appendQueryParams(url, {
+                [QUERY_PARAM_PDF_ENCODING]: PDFEncodingTypes.GZIP
```

Comment:
If the key/value pair is only used once, it can be declared inline. It doesn't look like we have any other encoding types.

Reply:
You could move this to before the docInitParams are set, then you can just pass this value in so you don't have to unpack it/modify a "const" object